### DISCUSSION TITLE
How do I say "No" to a confirmation prompt?

### DISCUSSION THREAD

**Discussion by aksh1618** [jesseduffield/lazygit#4014] [Q&A]
During a rebase, I'm getting the prompt: "Files have been modified since conflicts were resolved. Auto-stage them and continue?". This adds any files that weren't part of the git history to the rebase as well, which I don't want. Is there a way to say "No" to this prompt and continue with only the explicitly staged files?

**Comment by stefanhaller** [comment]
Hitting escape is the way to say No to this dialog and many similar ones.

**Reply by aksh1618** [reply]
I tried that, but it cancelled the rebase instead of proceeding without staging the additional files! Maybe I missed something, I'll try again and report back!

**Reply by stefanhaller** [reply]
Oh. Maybe we have a bug here. I'm on vacation right now and can't check; will have a closer look when I'm back home in a week.
